applications/app/views/fragments/indexBody.scala.html

```diff
@@ -3,10 +3,17 @@
 @import model.FrontProperties.empty
 @import services.IndexPage
 @import views.html.fragments.containers.facia_cards.container
+@import views.html.fragments.containers.facia_cards.containerScaffold
 @import views.html.fragments.pagination
 @import views.support.RenderClasses
 @import services.FaciaContentConvert
 
+@if(index.page.id == "help/accessibility-help") {
```

Comment:
seems a bit odd to have an if statement for all index pages just for this case, is it possible to have the dependency the other way to keep it out of the way?